ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -708,6 +708,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-public-private
+- as: aws-ipi-ovn-hypershift-public-private-p1-f21
+  cron: 37 0 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
```

Comment:
I'd like to know when will the tag `@critical` added, and when will not.

Reply:
For OCP release purpose, we will run full function test. No need to add @critical right now. 
For managed-hypershift (online service delivery), we will minimum test scope. @critical tag is needed here. 
In addition, we will optimize our pipeline in future if hypershift is stable on production. then we will consider some of jobs will just run @critical or @high to reduce resource cost.

Reply:
@liangxia ^^